server/views/pages/crmDetails.njk

```diff
@@ -57,7 +57,9 @@
                                             {% for field in subsection.fields %}
                                                 {% if field.subHeading %}
                                                     <div class="govuk-summary-list__row govuk-summary-list__row--no-border">
-                                                        <p class="govuk-brand-colour govuk-heading-s govuk-!-margin-top-{% if hasPreviousContent %}8{% else %}4{% endif %}">{{ field.subHeading }}</p>
+                                                        <p class="{% if section.sectionId != 'solicitors-declaration' %}govuk-brand-colour{% endif %} govuk-heading-s govuk-!-margin-top-{% if hasPreviousContent %}8{% else %}4{% endif %}">
```

Comment:
I don't think we should be checking for a particular sectionId here. Instead we should have a way to have subheadings which are blue or black.